packages/venia-ui/lib/util/images.js

```diff
@@ -17,23 +17,48 @@ export const imageWidths = {
     XXXLARGE: 2560
 };
 
-const generateURL = (imageURL, mediaBase) => (width, height) => {
-    const url = resourceUrl(imageURL, {
+const generateURL = (imageURL, mediaBase) => (width, height) =>
+    resourceUrl(imageURL, {
```

Comment:
Good suggestion.

On another note, do these functions return a full URL or just a pathname? I noticed that after you call them in `useProductImageCarousel`, you're prepending `origin` to their return value. If these happen to return pathnames rather than URLs, consider naming them accordingly.